### ISSUE TITLE
[ExpandCollapse] Update documentation and provide clear guidelines

### ISSUE THREAD

**Issue by donnavitan** [telus/tds-core#1095]
## Description
- Documentation does not provide clear guidelines on what content can go inside an expand/collapse
- Multiple instances where teams will want to use the expand and collapse component but were told not to use it near development/production time
- Not having clear guidelines wastes design effort early in project life cycle, and creates confusing and rush near the end of the life cycle
- Providing clear guidelines, in general content design, content hierarchy, or specific to component use, will help streamline the project life cycle and avoid any further delays to delivery

## Reproduction Steps
1. Create a design
2. Get approved
3. Start development
4. Get delayed due to incorrect use or design
5. Scramble to fix design

## Meta
- TDS component version: Version: 1.2.24
- Willing to develop solution: Yes (TDS)
- Has workaround: Yes, don't use component as per blocker discussion
- High impact: Yes, it affects how all teams use expand and collapse

## Screenshots
Original design (with expand and collapse):
![image](https://user-images.githubusercontent.com/21316148/58428286-016b1c00-8070-11e9-8cc9-cfda1d5291a4.png)

Revised design (without expand and collapse):
![image](https://user-images.githubusercontent.com/21316148/58428339-2495cb80-8070-11e9-8053-1ebb67f6034f.png)

https://telus.invisionapp.com/d/main#/console/16401801/360863493/preview
https://telus.invisionapp.com/d/main#/console/16401809/361050427/preview

## Next steps
- To provide clear documentation guidelines, we need to audit how expand and collapse is being used to determine patterns
- Once we have a collection of use cases, designs, screen captures, we can work with CX and design direction to determine the ideal customer experience to document guidelines

## How can you help?
- Please share, add, comment your use and needs for the use of the expand and collapse component.

**Comment by kittybourne** [comment]
![Screen Shot 2019-05-29 at 2 44 34 PM](https://user-images.githubusercontent.com/6434886/58593649-55b4fe00-8220-11e9-8180-ed768bc452b9.png)
![Screen Shot 2019-05-29 at 2 46 42 PM](https://user-images.githubusercontent.com/6434886/58593752-9dd42080-8220-11e9-9143-5b74d945b641.png)
FAQ's, tech specs

**Comment by DougTelus** [comment]
[Fitbit device page - Find a store](https://www.telus.com/en/bc/mobility/accessories/product/fitbit-charge-2)

![TELUS-Accessories-FITBIT-FindAStore-1A](https://user-images.githubusercontent.com/32107656/58986416-53681c00-87ab-11e9-8772-ef4819d659d9.png)
![TELUS-Accessories-FITBIT-FindAStore-1B](https://user-images.githubusercontent.com/32107656/58986423-57943980-87ab-11e9-8e7d-3da6d704a02e.png)

**Comment by twirlingsky** [comment]
Here are a few more from Home Solutions:
![Screen Shot 2019-06-11 at 10 23 07](https://user-images.githubusercontent.com/22402633/59293577-5001e400-8c34-11e9-9b26-a2e79aea5b35.png)
![Screen Shot 2019-06-11 at 10 25 27](https://user-images.githubusercontent.com/22402633/59293578-5001e400-8c34-11e9-8292-e824db341812.png)
![Screen Shot 2019-06-11 at 10 30 48](https://user-images.githubusercontent.com/22402633/59293579-5001e400-8c34-11e9-803b-192a468cdc3a.png)
![Screen Shot 2019-06-11 at 10 31 08](https://user-images.githubusercontent.com/22402633/59293580-5001e400-8c34-11e9-9328-468bd0ab9842.png)
![Screen Shot 2019-06-11 at 10 31 45](https://user-images.githubusercontent.com/22402633/59293581-5001e400-8c34-11e9-8c18-03612a0434de.png)

**Comment by Berjesty** [comment]
![Screen Shot 2019-06-11 at 3 33 48 PM](https://user-images.githubusercontent.com/25160096/59300914-602eb900-8c5e-11e9-99d3-5bf9ac11c776.png)
On Billing page we use chevron links inside expand collapse component.

**Comment by Berjesty** [comment]
On Billing page we use chevron links inside expand collapse component.
![Screen Shot 2019-06-11 at 3 34 01 PM](https://user-images.githubusercontent.com/25160096/59300976-86ecef80-8c5e-11e9-95fb-d183ff3c68e5.png)

**Comment by Berjesty** [comment]
On Overview page for business customers we have the underlined account selector treatment @apurvray pls feel free to comment further. 
![Screen Shot 2019-06-11 at 3 39 30 PM](https://user-images.githubusercontent.com/25160096/59301294-42ae1f00-8c5f-11e9-851b-70ac9b7f32a5.png)

**Comment by nikaukraine** [comment]
Few questions from Site Builder (Layout Grids - Row/Column perspective): 

1. What components should be available in Expand Collapse? We are thinking to allow everting from RichText: 
- Heading
- Paragraph
- Number list
- Bullet list
- Hairline divider
- Link

No images, buttons, chevron links etc? 

2. Should we allow to have more than 1 column within Expand Collapse?

**Comment by Chanmarylou** [comment]
<img width="548" alt="closed" src="https://user-images.githubusercontent.com/51763254/59384262-cb948b80-8d2f-11e9-848c-fe14adf30f49.PNG">
<img width="534" alt="open" src="https://user-images.githubusercontent.com/51763254/59384263-cb948b80-8d2f-11e9-9cb0-0eba71245073.PNG">

**Comment by DougTelus** [comment]
@Chanmarylou Can we have more context into the proposed design? Some rationale into what it is solving for, why it is better, reasons for +/-, keylines etc.

**Comment by Chanmarylou** [comment]
oops yes! Sorry I was just told to leave screenshots. I'm not picky about the +/-.  That's more of a design consideration- I was just looking for a collapsible block to use within SB.

**Comment by apurvray** [comment]
We are also using a custom expand-collapsed ( almost same design with some enhancements). I have attached a couple of screen shots below. 


![Screenshot 2019-06-13 14 29 08](https://user-images.githubusercontent.com/32710185/59458114-ce52b780-8de7-11e9-89a6-06a498a08d5a.png)
![Screenshot 2019-06-13 14 28 46](https://user-images.githubusercontent.com/32710185/59458115-ce52b780-8de7-11e9-8177-18473f902540.png)

**Comment by varunj90** [comment]
waiting on kitty to get back with direction from the content team: https://telusdigital.atlassian.net/browse/TDS-1360

**Comment by jraff** [comment]
Added Usage Criteria to ExpandCollapse. We will be creating a followup story to address FootnoteLinks in ExpandCollapse headers.
